service/apim-iati-dev-products.template.json

```diff
@@ -32,7 +32,7 @@
         "description": "IATI Technical Team approval is required. \n\nSubscribers have full access to all Public IATI APIs with the following limits:\n- Rate Limit: 5 API requests/1 second\n- Quota: 25,000 API requests/1 day\n    - Only successful HTTP response codes (200 - 399) are counted towards the quota.\n\nIf you reach the limit and believe you have a use case for higher limits, please contact support@iatistandard.org",
```

Comment:
We will need to update this description text too, but I don't mind if it happens in this PR.